docs/book/v6/tutorials/create-book-module.md

```diff
@@ -7,65 +7,103 @@ The below files structure is what we will have at the end of this tutorial and i
 ```markdown
 .
 └── src/
-    └── Book/
-        └── src/
-            ├── Collection/
-            │   └── BookCollection.php
-            ├── Entity/
-            │   └── Book.php
-            ├── Handler/
-            │   └── BookHandler.php
-            ├── InputFilter/
-            │   ├── Input/
-            │   │   ├── AuthorInput.php
-            │   │   ├── NameInput.php
-            │   │   └── ReleaseDateInput.php
-            │   └── BookInputFilter.php
-            ├── Repository/
-            │   └── BookRepository.php
-            ├── Service/
-            │   ├── BookService.php
-            │   └── BookServiceInterface.php
-            ├── ConfigProvider.php
-            └── RoutesDelegator.php
+    ├── Book/
+    │   └── src/
+    │       ├── Collection/
+    │       │   └── BookCollection.php
+    │       ├── Entity/
+    │       │   └── Book.php
+    │       ├── Handler/
+    │       │   ├── GetBookCollectionHandler.php
+    │       │   ├── GetBookHandler.php
+    │       │   └── PostBookHandler.php
+    │       ├── InputFilter/
+    │       │   └── CreateBookInputFilter.php
+    │       ├── Service/
+    │       │   ├── BookService.php
+    │       │   └── BookServiceInterface.php
+    │       ├── ConfigProvider.php
+    │       └── RoutesDelegator.php
+    ├── Core/
+    │   └── src/
+    │       └── Book/
+    │           └── src/
+    │               ├──Entity/
+    │               │   └──Book.php
+    │               ├──Repository/
+    │               │   └──BookRepository.php
+    │               └── ConfigProvider.php
+    └── App/
```

Comment:
Move `App` module above the `Book` module because that's how they appear in the IDE.